lib/archethic/networking/scheduler.ex

```diff
@@ -139,4 +130,19 @@ defmodule Archethic.Networking.Scheduler do
       {:ok, p2p_port, web_port}
     end
   end
+
+  defp handle_new_ip(%Transaction{address: tx_address}) do
+    nodes =
+      P2P.authorized_and_available_nodes()
+      |> Enum.filter(&Node.locally_available?/1)
```

Comment:
You can sort the node list by nearest node, so the await confirmations would be faster